Update iterator.js

maBna opened the pull request.

maBna committed:
Update iterator.js

maBna committed:
поправил codestyle

maBna commented:
А как снова проверить на соответствие кодстайлу? А то при пулл реквесте автоматом проверилось, а сейчас нет.

maBna committed:
Снова поправил codestyle

maBna committed:
И еще раз поправил codestyle

maBna committed:
Update iterator.js

Поправил ошибку с глубиной
Теперь next принимает что угодно и по этому параметру возвращает следующего

maBna committed:
Убрал нелегальный пробел

i4got10 commented:
Есть проблемы с обходом графа, местами порядок неправильный. Вот глянь картинку

![js4-serg](https://cloud.githubusercontent.com/assets/1151154/10817449/aa3955f6-7e5b-11e5-9f95-875def10b81a.png)

maBna committed:
Небольшие правки

Теперь не крашится при пустой коллекции и некст и прев работают нормально, если вылезаем за края списка друзей

maBna committed:
Правка

maBna committed:
Поправил кодстайл

maBna committed:
Еще правка

Поправил вывод нулл, если нет в коллекции стартовой точки.

maBna committed:
Еще исправление

maBna committed:
Кодстайл

mokhov commented:
Принято
